tests/account_test.js

```diff
@@ -0,0 +1,12 @@
+Feature('Guestbook');
+    const r_ship = Math.floor(Math.random() * 5) + 1
```

Comment:
All const should be defined above "Feature". The only thing that should be below "Feature" is the tags attached to it and the scenarios.

Reply:
thanks, I will make this update.

Reply:
@ghopkins-legacy the PR has been updated based on your comment